java/spacewalk-java.spec

```diff
@@ -666,14 +666,21 @@ chown tomcat:%{apache_group} /var/log/rhn/gatherer.log
 %{jardir}/dom4j.jar
 %{jardir}/dwr.jar
 
-%{jardir}/google-gson.jar
 %{jardir}/snakeyaml.jar
 # SUSE extra runtime dependencies: spark, jade4j, salt API client + dependencies
 %{jardir}/commons-jexl.jar
 %{jardir}/commons-lang3.jar
+%if 0%{?is_opensuse}
+%{jardir}/google-gson_google-gsongson.jar
+%{jardir}/httpcomponents_httpclient.jar
+%{jardir}/httpcomponents_httpcore.jar
+%{jardir}/httpcomponents_httpcore-nio.jar
```

Comment:
Is this really correct? I would have expected an "/" instead of "_" here. But I'm not a java developer :)

Reply:
The name of the symlinks is generated automatically by `build-jar-repository` from the directory name + the jar name.

Reply:
I was expecting something like that...